howto/How_To_Use_OpenOCD_UbuntuEd

```diff
@@ -0,0 +1,67 @@
+---
+title: Openocd On Ubuntu 64
+redirect_from:
+- /Openocd.html
+---
+This How to covers, if I rip my usb how do I flash the Proffieboard or update it. there's a few things about this particular use case. In this article you will be using the ST-Linkv2 usb debugger that connects through single core and the operating system of your laptop or pc would be Ubuntu 64 bit or any linux distro that is 64 bit. 
```

Comment:
Overall, the structure of the text needs some work.
It reads more like "this is what I did" than "this is what you should do".